### ISSUE TITLE
Request: add ability to "refresh" or "reload" current file while keeping position, marks, search highlight, and other settings

### ISSUE THREAD

**Issue by jolsen71** [gwsw/less#136]
Please add the ability to "refresh" or "reload" current file while keeping position, marks, search highlight, and other settings that may have been altered/set since less was launched to view the file. 

Less is one of my most often used tools, and I will sometimes use it to look at code from another developer on a remote server, and sometimes it is while they are working on the code so I can see their updates and provide real-ish-time feedback. It would be pretty nice to be able to reload/refresh the contents of a file from within less and being able to maintaining the current state within the less session in regard to position in file, line chopping, highlighted search terms, hidden/visible line numbers, marks, etc. instead of having to exit less, re-less the file, re-navigate back to the point in the file I was at, re-turn-on line numbers (they're off for me by default, which is what I want), etc.

In addition to a manual refresh (a key command), it would be a bonus to be able to enable/disable auto-refresh every x seconds from within less (perhaps in a similar way to "forward forever"/follow mode where it has to be disabled before you can anything else with less)

Thanks for listening

**Comment by gwsw** [comment]
Doesn't the R command do what you want?

**Comment by gwsw** [comment]
In b7ce5cdbc20151ca6e1fe9238e04a8617b54795a I added a line to the description of the R command that should help.

**Comment by jolsen71** [comment]
I looked at the b7ce5cd  diff, and saw there was already a line in `less.nro.VER` that said "Useful if the file is changing while it is being viewed" and I hadn't remembered seeing that. I opened `less`, hit `H` for help, which is how I was originally trying to find this function, and noticed that text wasn't there. Since it had to be somewhere, I figured it must be in the man page so I ran `man less` and sure enough, that's where it is visible.

For people that use the in-program help like I was doing, it may still be worth considering changing the current line 111 in `less.nro.VER` to something like:
`Reload file (Repaint the screen, discarding any buffered input)`
or if it is too long, perhaps something like:
`Reload file (Repaint screen, discarding buffered input)`

Thanks again for a fast response and great tool.
